libafl/src/bolts/shmem.rs

```diff
@@ -1397,22 +1397,22 @@ impl<T: ShMem> std::io::Seek for ShMemCursor<T> {
             std::io::SeekFrom::Start(s) => s,
             std::io::SeekFrom::End(offset) => {
                 let map_len = self.inner.as_slice().len();
-                assert!(i64::try_from(map_len).is_ok());
+                assert!(i64::try_from(map_len).unwrap());
                 let signed_pos = map_len as i64;
                 let effective = signed_pos.checked_add(offset).unwrap();
                 assert!(effective >= 0);
                 effective.try_into().unwrap()
             }
             std::io::SeekFrom::Current(offset) => {
                 let current_pos = self.pos;
-                assert!(i64::try_from(current_pos).is_ok());
+                assert!(i64::try_from(current_pos).unwrap());
                 let signed_pos = current_pos as i64;
                 let effective = signed_pos.checked_add(offset).unwrap();
                 assert!(effective >= 0);
                 effective.try_into().unwrap()
             }
         };
-        assert!(usize::try_from(effective_new_pos).is_ok());
+        assert!(usize::try_from(effective_new_pos).unwrap());
```

Comment:
Unwrapping doesn't need an assert

Reply:
(right?)

Reply:
clippy suggested this silly thing! I'll just add allow()s